src/dealer-details.js

```diff
@@ -44,9 +44,12 @@ const DealerDetails = ({ dealer, close, closeButton, websiteButton }) => {
           <DealerDetailRow>
             <DealerAddress>
               <div>{dealer.addr1}</div>
-              <div>{`${dealer.city} ${dealer.city ? "," : ""} ${dealer.state} ${
-                dealer.zip
-              } ${dealer.country}`}</div>
+              <div>
+                {dealer.city ? `${dealer.city}, ` : ""}
+                {dealer.state ? `${dealer.state}, ` : ""}
```

Comment:
It looks like this would also have a change of adding an extra comma after state, which isn't the usual formatting

Correct: 
Salt Lake City, UT 84105

Incorrect
Salt Lake City, UT, 84105

Reply:
Good catch. `Salt Lake City, UT 84105, USA` or `Salt Lake City, UT 84105 USA`?

Reply:
That one I'm not sure about. Might be worth just sticking a story at the top of the backlog so we can get sign off on the requirements before going to the trouble of updating this across repos